Assets/Scripts/Properties/LoadWindows.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using System.Runtime.CompilerServices;
+
+public class LoadWindows : MonoBehaviour
+{
+    [SerializeField] private GameObject _levelSelection         = null;
+    [SerializeField] private GameObject _characterCustomization = null;
+    [SerializeField] private GameObject _familySetup            = null;
+    [SerializeField] private GameObject _characterSetup         = null;
+    [SerializeField] private GameObject _pauseMenu              = null;
+    [SerializeField] private GameObject _completionMenu         = null;
+    [SerializeField] private GameObject _shopMenu               = null;
+
+    private void Awake()
+    {
+        Properties.windows = new GameObject[7]
+        {
+            _levelSelection,
+            _characterCustomization,
+            _familySetup,
+            _characterSetup,
+            _pauseMenu,
+            _completionMenu,
+            _shopMenu
+        };
```

Comment:
These GameObjects seem to correspond to the same window types as defined in the WindowTypes enumerator.
Hardcoding every object like this leaves no room for adjustments and the same result could be achieved if the GameObject array is filled in the inspector using a serialized array, List or even Dictionary that uses the corresponding enum type as key, rather than having all GameObjects serialized separately.
Furthermore, none of the objects seems to be ever used after being put in an array.